go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/stackrox/helmtest
 
-go 1.22
+go 1.22.1
 
 toolchain go1.22.5
 
 require (
-	github.com/golangci/golangci-lint v1.55.2
```

Comment:
I think we should pull linter out of the main module into tools like we have in other projects

Reply:
https://github.com/stackrox/helmtest/pull/155